readme.txt

```diff
@@ -1,18 +1,16 @@
 === WP LCache ===
 Contributors: getpantheon, danielbachhuber, stevector
-Tags: cache, plugin
+Tags: apc, apcu, cache, caching, lcache, object-cache, performance
 Requires at least: 4.3
-Tested up to: 4.7.4

```

Comment:
@danielbachhuber Because WordPress ignores minor releases in this field so you aren't required to bump your versions at every point release, making it just added verboseness.

https://plugins.svn.wordpress.org/jetpack/trunk/readme.txt
https://plugins.svn.wordpress.org/woocommerce/trunk/readme.txt
https://plugins.svn.wordpress.org/wordpress-seo/trunk/readme.txt